webapp/src/actions/vote.js

```diff
@@ -0,0 +1,38 @@
+import {doPostAction} from 'mattermost-redux/actions/posts';
+
+import {id as pluginId} from 'manifest';
+import ActionTypes from 'action_types';
+
+export const voteAnswer = (postId, actionId) => async (dispatch) => {
+    return dispatch(doPostAction(postId, actionId));
+};
+
+export const websocketHasVoted = (data) => async (dispatch) => {
+    return dispatch({
+        type: ActionTypes.FETCH_VOTED_ANSWERS,
+        data: {
+            user_id: data.user_id,
+            poll_id: data.poll_id,
+            voted_answers: data.voted_answers,
+        },
+    });
+};
+
+export const fetchVotedAnswers = (siteUrl, pollId) => async (dispatch) => {
+    if (typeof pollId === 'undefined' || pollId === '') {
```

Comment:
could instead be `if (Boolean(pollId))`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean

Reply:
I didn't know this func, thanks! However `Boolean` function was marked as not recommended by ESLint.
https://eslint.org/docs/2.0.0/rules/no-extra-boolean-cast

So I choose `if (!pollId)` simply. Are you fine with this?

Reply:
Strange as I have used (and was recommend to) use Boolean in the mattermost-webapp which also uses ESlint.  I am OK with the simple (!pollId). :)

Reply:
Ah...it may be caused by the difference of `.eslintrc.json`. It would be better to make settings of ESLink the same as mattermost-webapp's one.

However, it seems that a warning about Boolean function will occur even if the settings are the same as mattermost-webapp ... I'll take a look more.

Reply:
I understand. ESLint reports `no-extra-boolean-cast` when using `Boolean` function in `if` condition.  So here `if (!pollId) {` looks good.